packages/office-ui-fabric-react/src/components/TextField/examples/TextField.CustomRender.Example.tsx

```diff
@@ -18,7 +19,7 @@ export class TextFieldCustomRenderExample extends React.Component<any, any> {
 
   public render() {
     return (
-      <div>
+      <div className='ms-TextFieldExample'>
```

Comment:
I know this is being done everywhere in the documentation, so this is probably not the time to fix it, but the `ms-` prefix should only be used on actual Fabric components that we distribute. So classes like this could be `docs-TextFieldExample` or simply `TextFieldExample` to distinguish between Fabric components and documentation-specific classes.